apps/ledger-live-desktop/src/renderer/components/PostOnboardingHub/index.tsx

```diff
@@ -11,37 +9,22 @@ import { withV3StyleProvider } from "~/renderer/styles/StyleProviderV3";
 
 const PostOnboardingHub = () => {
   const dispatch = useDispatch();
-  const { t } = useTranslation();
-  const history = useHistory();
   const { lastActionCompleted, actionsState } = usePostOnboardingHubState();
   const clearLastActionCompleted = useCallback(() => {
     dispatch(clearPostOnboardingLastActionCompleted());
   }, [dispatch]);
-  const isInsidePostOnboardingScreen = history.location.pathname === "/post-onboarding";
-
-  const handleStartAction = useCallback(action => {
-    action.startAction();
-  }, []);
 
   return (
     <Flex flexDirection="column" justifyContent="center">
-      {actionsState.map((action, index, arr) => (
-        <React.Fragment key={index}>
-          <Box
-            onClick={() => {
-              if (action.navigationParams) history.push(...action.navigationParams);
-              else if (action.startAction) {
-                if (isInsidePostOnboardingScreen) {
-                  history.push("/");
-                }
-                handleStartAction(action);
-              }
-            }}
-          >
-            <PostOnboardingActionRow {...action} />
-          </Box>
-        </React.Fragment>
-      ))}
+      {useMemo(
```

Comment:
⚠️ never call a hook inside of some JSX code.
Declare this as a variable before the return 👍